### ISSUE TITLE
get_index() funtion returns 0('[CLS]') as an index

### ISSUE THREAD

**Issue by stephen-pilli** [keitakurita/contextual_embedding_bias_measure#1]
In
 
>weat_result_replication_permute_targets.ipynb

There seems to be problem with get_index() called in function in get_mask_fill_logits.

```
print(np.log(get_mask_fill_logits("[MASK] are beautiful.", ["clover"])['clover']))
print(np.log(get_mask_fill_logits("[MASK] are [MASK].", ["clover"], use_last_mask=True)['clover']))
```
results in 

gender_comes_first=False
>Tokens for ptgt :  [[MASK], are, beautiful, .]
	 0 [MASK]
return maskid from get_index() function :  1
response from get_mask_fill_logits : -18.595278

gender_comes_first=True
>Tokens for prior : [[MASK], are, [, [UNK]]
	 0 [UNK]
	 1 [
	 2 are
	 3 [MASK]
maskid :  0
response from get_mask_fill_logits : -10.874125

gender_comes_first=True
Expected response:
>Tokens for prior : [[MASK], are, [MASK], .]
	 0 [MASK]
	 1 are
	 2 [MASK]
	 3 .
maskid :  3
response from get_mask_fill_logits : -28.40166
